src/coreclr/jit/importer.cpp

```diff
@@ -3673,37 +3674,62 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
         assert(!info.compMatchedVM);
     }
 
-    ni = lookupNamedIntrinsic(method);
-
     // We specially support the following on all platforms to allow for dead
     // code optimization and to more generally support recursive intrinsics.
 
-    if (ni == NI_IsSupported_True)
+    if (isIntrinsic)
```

Comment:
Why are we suppressing these for altjit? I would expect that NI_IsSupported_True/NI_IsSupported_False is going to be accurate even under altjit.

Reply:
All of the `IsSupported` APIs that should be handled are marked `[Intrinsic]` on all platforms already

Any other APIs that might get flagged would be incorrect and can lead to the JIT performing incorrect optimizations.

Reply:
Ok, make sense.